sdcm/cluster.py

```diff
@@ -5390,12 +5390,19 @@ def install_scylla_monitoring_prereqs(self, node):  # pylint: disable=invalid-na
 
         if node.distro.is_rhel_like:
             node.install_epel()
-            node.install_package(package_name="unzip wget python36 python36-pip")
-            prereqs_script = dedent("""
-                python3 -m pip install --upgrade pip
-                python3 -m pip install pyyaml
+            node.install_package(package_name="unzip wget python3 python3-pip")
+            # get-docker.sh doesn't support Rocky, thus docker should be installed from repo via dnf install
+            install_docker_cmd = dedent("""
+                dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+                dnf install -y docker-ce docker-ce-cli containerd.io
+            """ if node.distro.is_rocky else """
```

Comment:
Can we not inline this if ? So it would be a bit more readable, and assign to the variable as needed ?